Source/WebCore/dom/Element.h

```diff
@@ -131,6 +131,8 @@ class Element : public ContainerNode {
     WEBCORE_EXPORT void setElementAttribute(const QualifiedName& attributeName, Element* value);
     WEBCORE_EXPORT std::optional<Vector<RefPtr<Element>>> getElementsArrayAttribute(const QualifiedName& attributeName) const;
     WEBCORE_EXPORT void setElementsArrayAttribute(const QualifiedName& attributeName, std::optional<Vector<RefPtr<Element>>>&& value);
+    WEBCORE_EXPORT static bool isElementReflectionAttribute(const QualifiedName&);
```

Comment:
Do we really need to export these two symbols???

Reply:
Yes, we don't need `WEBCORE_EXPORT` for these two. Removed it.